tests/unit/test_schema_utils.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+import pytest
+from typing import Any
+
+from schematic.schemas.data_model_parser import DataModelParser
+from schematic.utils.schema_utils import parsed_model_as_dataframe, check_allowed_values
+from schematic.schemas.data_model_relationships import DataModelRelationships
+
+
+class TestSchemaUtils:
+    """
+    Tests for the schema utils module.
+    """
+
+    def test_model_as_dataframe(self) -> None:
+        """
+        Tests DataModelParser.model_as_dataframe
```

Comment:
nit: I couldn't find `model_as_dataframe`. Perhaps you meant `parsed_model_as_dataframe`?